partiql-ast/src/main/kotlin/org/partiql/ast/sql/Sql.kt

```diff
@@ -1,15 +1,41 @@
 package org.partiql.ast.sql
 
 import org.partiql.ast.AstNode
+import org.partiql.ast.sql.internal.InternalSqlDialect
+import org.partiql.ast.sql.internal.InternalSqlLayout
+
+/**
+ * No argument uses optimized internal. Leaving older ones for backwards-compatibility.
+ */
+public fun AstNode.sql(): String {
+    val head = InternalSqlDialect.PARTIQL.apply(this)
+    return InternalSqlLayout.format(head)
+}
 
 /**
  * Pretty-print this [AstNode] as SQL text with the given [SqlLayout]
  */
-@JvmOverloads
+@Deprecated("To be removed in the next minor version")
```

Comment:
For these `@Deprecated` messages, should they say next major version?

```suggestion
@Deprecated("To be removed in the next major version")
```